tests/microos/verify_setup.pm

```diff
@@ -300,6 +304,15 @@ sub run {
     file_tests();
     disk_tests();
 
+    if (get_var('FIRST_BOOT_CONFIG') =~ /combustion/ && !(is_sle_micro('<5.5') || is_leap_micro('<5.5'))) {
```

Comment:
It is confirmed that SLE Micro 6.0 will have this feature, I guess the same applies for Dolomite.

Reply:
for sure, it is already in sle-micro 5.5